src/pages/messaging/agent-chat-api/v3.3/index.mdx

```diff
@@ -1172,7 +1172,6 @@ To call the API as a bot, use the `X-Author-Id` header set to the bot's id.
 | ----------------------- | -------- | --------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | `chat_id`               | Yes      | `string`  | Id of the chat you want to send a message to.                                                                                                                     |
 | `event`                 | Yes      | `object`  | [Event](/messaging/agent-chat-api/data-structures/#events) object. Does not support the `form` type event in the LiveChat app.                                    |
-| `attach_to_last_thread` | No       | `bool`    | The flag is ignored for active chats. Default: `false`. For inactive chats: `true` – the event will be added to the last thread; `false` – the request will fail. The only event type that can be sent to inactive chats is a system message. |
```

Comment:
The flag is working correctly in v3.3, so there's no need to remove it from here.

The reason it stopped working from v3.4+ is that we started requiring the requester to be added as a chat user, which is not possible for inactive chats. It was introduced in v3.4+, so maybe we should mention it somehow in the changelog 🤔